src/web/src/pages/CoursePage.vue

```diff
@@ -70,10 +62,25 @@ export default {
     this.ready = true;
   },
   computed: {
-    prereqTree() {
-      let treeData = {};
-      treeData.name = this.courseName;
-      return treeData;
+    transformed() {
+      let precoreqtext = this.courseObj.raw_precoreqs;
+      if (precoreqtext === null) {
+        return "This course has no prerequisites";
```

Comment:
Maybe we change this line over to "No information on pre/corequisites" wyt?

Reply:
a little non-committal ;) No thats much better